docs-source/spring/content/development/cli/_index.md

```diff
@@ -25,19 +23,80 @@ Table of Contents:
   * [List - show details about deployments](#list)
   * [Manage config server data](#config)
   * [GraalVM compile commands](#compile)
+  * [User Management](#user-management)  
+* [Logging Information](#logging)
+
+## Installation
+
+1. CLI is available for Linux on x64 and AArch64, macOS on x64 and AArch64, and Windows x64 architectures. Choose your operating system and proceed to the installation steps for your specific platform:
+
+   * [Linux x86](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-linux-x86)
+   * [Linux AArch64](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-linux-aarch64)
+   * [macOS x86](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-linux-x86)
+   * [macOS AArch64](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-macos-aarch64)
+   * [Windows x86](https://github.com/oracle/microservices-datadriven/releases/download/OBAAS-1.1.0/oractl-1.1-windows-x86)
+
+2. Change to directory where you want to install CLI. The platform-specific binary can be renamed to `oractl` for convenience.
+
+    ```cmd
+    ❯ mv oractl-<version>-<os>-<architecture> /path/to/<obaas>/oractl
+    ```
+  
+    &nbsp;
+
+    ---
+
+    {{< hint type=[warning] icon=gdoc_info_outline
+        title="Remove the quarantine attribute (Required for macOS Catalina and later)" >}}
+    {{< /hint >}}
+
+    ```cmd
+    ❯ sudo xattr -r -d com.apple.quarantine /path/to/<obaas>/oractl
+    ```
+
+    ---
+
+3. In Linux-like systems, you must adjust the file permissions from `oractl` to allow its execution.
+
+    ```cmd
+    ❯ chmod a+x /path/to/<obaas>/oractl
+    ```
+
+4. Set the value of the `PATH` environment variable to the `oractl` directory:
+
+    ```cmd
+    ❯ export PATH=/path/to/<obaas>:$PATH
+    ```
 
 ## Using the CLI
 
-1. Expose the Oracle Backend for Spring Boot and Microservices Admin server that the CLI calls using this command:
+Once you have installed the CLI, you can run it by typing `oractl` and pressing Enter at the command line. But first you have to open a bridge with the with Oracle Backend for Spring Boot and Microservices Admin Service. Follow the next steps to run `oractl`.
```

Comment:
don't tell people they need to press enter to run a command, that is obvious. 
don't start a sentence with "but"
put the steps in the order they need to be executed - tell them about the tunnel first, then the oractl execution

Reply:
its a "tunnel" not a "bridge"